src/content/docs/dropins-b2b/purchase-order/containers/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Purchase Order Containers
+description: Overview of containers available in the Purchase Order drop-in.
+sidebar:
+  label: Overview
+  order: 1
+---
+
+import { Aside } from '@astrojs/starlight/components';
+import TableWrapper from '@components/TableWrapper.astro';
+
+The **Purchase Order** drop-in provides **12** pre-built container components for integrating into your storefront.
```

Comment:
I know AI is doing the counting and might even notice it if a new container becomes available, but I don't like giving a specific number.

Reply:
Yeah, I can remove that. I think that's a fairly unanimous sentiment. It's not necessary. Devs don't care.